### DISCUSSION TITLE
Linux Driver

### DISCUSSION THREAD

**Discussion by opnine** [carlmontanari/scrapli#257] [General]
Hi all , 
I was working with Nornir / Scrapli but now I started getting the error " Scrapli Community platform 'linux` not found!" Any way I can fix this? thanks. !

**Comment by carlmontanari** [comment]
use the generic driver https://carlmontanari.github.io/scrapli/user_guide/advanced_usage/#using-the-genericdriver

**Reply by opnine** [reply]
Thanks!, and for Platform will still use Generic? 
thanks.
